pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -199,6 +199,9 @@
 
     public static final String BROKER_TLS_PREFIX = "pinot.broker.tls";
     public static final String BROKER_NETTYTLS_ENABLED = "pinot.broker.nettytls.enabled";
+    public static final String BROKER_DYNAMIC_HELIX_HOST = "pinot.broker.dynamic.helix.host";
+    public static final String BROKER_NETTY_PORT = "pinot.broker.netty.port";
+    public static final String BROKER_NETTY_HOST = "pinot.broker.netty.host";
```

Comment:
We don't need these extra configs. Check `HelixBrokerStarter` constructor which already has host and port info is available

Reply:
let me double check that; forgot this comment

Reply:
Removed now